### PR TITLE
[imageio] Update for Xcode11 beta 7

### PR DESCRIPTION
**Pull request by spouliot**

### CODE REVIEW

**Comment by spouliot** [comment]
CGImageAnimation.h not included in ImageIO umbrella header
https://github.com/xamarin/maccore/issues/1941
https://feedbackassistant.apple.com/form-response/15197631 (issue submitting)

File: src/imageio.cs [hunk 1]
```diff
@@ -306,6 +310,18 @@ interface CGImageProperties {
 		[Field ("kCGImagePropertyGIFUnclampedDelayTime")]
 		NSString GIFUnclampedDelayTime { get; }
 
+		[iOS (13,0)][TV (13,0)][Watch (6,0)][Mac (10,15)]
+		[Field ("kCGImagePropertyGIFCanvasPixelWidth")]
+		NSString GifCanvasPixelWidth { get; }
```
Comment (rolfbjarne) [inline src/imageio.cs]: All the other fields here use `GIF` and not `Gif`.

Comment (spouliot) [reply src/imageio.cs]: yes, I'm a bit ambivalent between old stuff and current rules :)

Comment (spouliot) [reply src/imageio.cs]: Code completion does not care about casing (it's still alphabetical order) so I tend toward keeping the new stuff following the newer rules (i.e. it's not inconsistent in the usage, only the look).

**Pull request merged**